addons/website_slides/views/slide_channel_views.xml

```diff
@@ -15,9 +15,8 @@
                             <button name="action_view_slides"
                                 type="object"
```

Comment:
You can also remove the `name` and `type` attribute if it's no longer callable :smiley:

Reply:
Hello :slightly_smiling_face: 

The thing is, if I remove the name and type attribute the stat button looks like it was disabled:
![image](https://user-images.githubusercontent.com/107679666/216995273-98df765c-52b8-4e33-879b-9437dc9297bf.png)
instead of keeping it's style like so:
![image](https://user-images.githubusercontent.com/107679666/216995502-11c88882-4d35-4997-891f-a159178486ca.png)

It looks a bit weird for it to be displayed as disabled don't you think? :thinking: 
I'll ask lna about it to be sure :slightly_smiling_face:

Reply:
Indeed, it looks better when it's not displayed as a disabled button :astonished:

Reply:
Hello :slightly_smiling_face: 
Actually, frgi want it to look disabled so I just removed the name and type attributes, but also the disabled attribute which wasn't necessary for it to look disabled as there is no action to trigger anymore.

Cheers! :slightly_smiling_face: